managers/templates/managers/project_form.html

```diff
@@ -6,6 +6,8 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
+
```

Comment:
Extra empty lines are not needed.

Reply:
Yeah, it's a styling error on my part. I just didn't have any time left to fix that.